lib/init.coffee

```diff
@@ -1,14 +1,67 @@
+{BufferedProcess, CompositeDisposable} = require 'atom'
+
 module.exports =
   config:
-    elixircExecutablePath:
-      type: 'string'
-      default: ''
-    includeDirs:
-      type: 'string'
-      default: ''
-    pa:
+    executablePath:
       type: 'string'
-      default: ''
+      title: 'Elixirc path'
+      default: '/usr/local/bin/elixirc'
 
   activate: ->
-    console.log 'activate linter-elixirc'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-elixirc.executablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+  deactivate: ->
+    @subscriptions.dispose()
+  provideLinter: ->
+    provider =
+      grammarScopes: ['source.elixir']
+      scope: 'file'
+      lintOnFly: false
+      name: 'Atom-Elixir'
+      lint: (textEditor) =>
+        return new Promise (resolve, reject) =>
+          error_stack = []
+          compile_result = ''
+          helpers = require('atom-linter')
```

Comment:
Can you move these requires one level up?
I don't want them to be imported on every lint call. They stay the same.